packages/rocketchat-mentions/tests/client.tests.js

```diff
@@ -132,16 +132,17 @@ describe('Mention', function() {
 				'hello #general, how are you?'
 			].forEach(text => {
 				it(`should return "${ JSON.stringify(result) }" from "${ text }"`, () => {
+					console.log('result', mention.getChannelMentions(text));
```

Comment:
Did you check if this console.log breaks any test?

Reply:
It doesn't break, but I'll remove.